src/main/resources/assets/lambdabettergrass/lang/ru_ru.json

```
@@ -0,0 +1,12 @@
+{
+  "lambdabettergrass.menu.info.1": "Мод использует ресурспак, чтобы определить, какие блоки затронуты.",
+  "lambdabettergrass.menu.info.2": "Включение или отключение функции улучшенного дёрна/снега на блоках",
+  "lambdabettergrass.menu.info.3": "нужно модифицировать с помощью ресурспака.",
+  "lambdabettergrass.menu.info.read_more": "Подробнее читайте на: %s",
+  "lambdabettergrass.menu.title": "Настройки LambdaBetterGrass",
+  "lambdabettergrass.menu.title.info": "Как работает мод?",
+  "lambdabettergrass.option.better_snow": "Улучшенный снег",
+  "lambdabettergrass.option.mode": "Улучшенный дёрн",
```

```suggestion
  "lambdabettergrass.option.mode": "Улучшенная трава",
```
"Дёрн" – название всего блока в игре, мод же изменяет лишь текстуру травы. Поэтому предлагаю использовать слово "Трава" везде, где речь идёт только о текстуре.